### DISCUSSION TITLE
About image optimization

### DISCUSSION THREAD

**Discussion by csaunier** [caching-tools/next-shared-cache#314] [Q&A]
Hello :wave: 

I would like to now if the cache management through `nextjsconfig.cacheHandler` or `nextjsconfig.experimental.incrementalCacheHandlerPath` also handle the caching of the optimized image through `next/img`, or if this is another mechanism to implement ?

Thanks !!

**Comment by better-salmon** [comment]
Hello, @csaunier! Custom cache handlers do not process images in any way. You may use special tools such as https://imgproxy.net/ or load them to CDN on build.

**Comment by ezeparziale** [comment]
> Hello 👋
> 
> I would like to now if the cache management through `nextjsconfig.cacheHandler` or `nextjsconfig.experimental.incrementalCacheHandlerPath` also handle the caching of the optimized image through `next/img`, or if this is another mechanism to implement ?
> 
> Thanks !!

Check https://nextjs.org/docs/app/building-your-application/deploying#image-optimization
